src/redux/actions/planActions.js

```diff
@@ -256,6 +256,7 @@ export function getPlansFromDb() {
  * @returns {function(*): Promise<Response | never>} dispatch results.
  */
 export function addPlanToDb(plan) {
+  console.log('Redux', plan);
```

Comment:
Should probably remove this :)

Reply:
Hah. Yeah, thought I got all the logs I was using... I think I may begin commenting `//TODO remove`s next to all of my temporary logs while I work. Thanks for catching this.